frontend/packages/dev-console/src/components/import/import-types.ts

```diff
@@ -9,10 +9,32 @@ export interface DeployImageFormProps {
   imageStreams?: FirehoseList;
 }
 
-export interface ImageStreamProps {
-  imageStreams: K8sResourceKind[];
+export type ImageStreamState = {
+  hasAccessToPullImage: boolean;
+  accessLoading: boolean;
+  loading: boolean;
+  hasCreateAccess: boolean;
+  selectedImageStream: K8sResourceKind | any;
```

Comment:
Do we need type ```any``` here. Isn't ```selectedImageStream``` is always of type ```K8sResourceKind```.

Reply:
Again, this is added due to the lint error which happened for union of types. I fixed it along with the below comment.